src/images/icons/add_more.svg

```diff
@@ -1,7 +1,4 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512">
-  <title>
-    add more
-  </title>
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" aria-labbeledby="add more">
```

Comment:
this should not be `aria-labelledby` (btw. there's a typo here) but `aria-label`. [aria-labelledby](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-labelledby) needs another item with the id provided between "" to be able to label this element

Reply:
That what's happening when you blindly copy paste from the docs without thinking 😄 Yes I know, I will change it, thanks!

Reply:
but to be honest I'm not sure if the svg does actually need any label - it just has a presentational role. I'd suggest to give it `alt=""` as this suggests to the screen readers that it doesn't matter for the page content